README.md

```diff
@@ -50,7 +71,9 @@ func init() {
 				Panic: 13631488,
 				Fatal: 13631488,
 			},
-			DisableInlineFields: false, // If set to true, fields will not appear in columns ("inline")
+			DisableTimestamp: false,           // Setting this to true will disable timestamps from appearing in the footer
```

Comment:
Can we keep the old ordering (although `TimestampLocale` should still be directly after `TimestampFormat`)? It's more likely for time-related details to be changed than level colours, so time-related fields should take precedent.

Reply:
✔